library/kotlin/src/io/envoyproxy/envoymobile/ResponseHeaders.kt

```diff
@@ -14,7 +14,9 @@ class ResponseHeaders : Headers {
   /**
    * HTTP status code received with the response.
    */
-  val httpStatus: Int? by lazy { value(":status")?.first()?.toIntOrNull() }
+  val httpStatus: Int? by lazy {
```

Comment:
Should this be a UInt?

Reply:
Ah, sorry this is the Kotlin one. Got it.

Reply:
Kotlin doesn't have the concept of unsigned integers available to us yet: https://kotlinlang.org/docs/basic-types.html#unsigned-integers, which is why we have a unit test for this now. Meant to mention that in the PR but forgot